src/lib.rs

```diff
@@ -135,6 +135,10 @@
 #![deny(rustdoc::broken_intra_doc_links)]
 #![deny(clippy::all)]
 #![cfg_attr(feature = "cargo-clippy", deny(warnings))]
+// Enable doc_auto_cfg when generating docs.
```

Comment:
The user can read themselves that it is being enabled when the `docsrs` cfg is true, no need to paraphrase the code here.

Why don't you explain what it means/does instead? Otherwise, remove this line altogether.